cloud/terraform/tpot.conf

```diff
@@ -0,0 +1,5 @@
+# tpot configuration file
+# myCONF_TPOT_FLAVOR=[STANDARD, SENSOR, INDUSTRIAL, COLLECTOR, NEXTGEN, LEGACY]
```

Comment:
Just noticed there is no `LEGACY` edition any longer. Can you adjust this please?

Reply:
Sure. I used [tpot.conf.dist](https://github.com/dtag-dev-sec/tpotce/blob/master/iso/installer/tpot.conf.dist) as an example though. I guess this should be changed as well?

Is this correct?

```
# tpot configuration file
# myCONF_TPOT_FLAVOR=[STANDARD, SENSOR, INDUSTRIAL, COLLECTOR, NEXTGEN]
myCONF_TPOT_FLAVOR='STANDARD'
myCONF_WEB_USER='webuser'
myCONF_WEB_PW='w3b$ecret'
```

Reply:
Thank you, missed that and will take care of it :)

Reply:
No problem. I've updated on my side as well.